### ISSUE TITLE
Proposed: obolibrary.org for the new ontology listing site, obofoundry.net for foundry description with link to obolibrary

### ISSUE THREAD

**Issue by alanruttenberg** [OBOFoundry/OBOFoundry.github.io#127]
Title tries to say it all. The goal is to differentiate Foundry from the library in general, which includes non-foundry. A simple way to do this is to have two sites, one introducing the foundry, with a link to the ontology listing, and one for the ontology listing itself. I own domains obolibrary.net/org/com and will happily donate them.

**Comment by cmungall** [comment]
It might be hard to tease apart the content. Or are you thinking the entirety of the foundry page might be something like a single splash page, with the majority of the content continuing to be colocalized in one place, now called obolibrary.org?

Another way to resolve confusion is to go back to the original foundry concept (https://en.wikipedia.org/wiki/Foundry_%28disambiguation%29) as a kind of plant for producing certain kinds of product. You would expect an actual foundry to have all kinds of yet-to-be-formed blobs. Seeing the obofoundry website perhaps should be like touring an actual foundry, seeing the full production line.

In fact in an actual foundry the finished fully formed products would not stay in the foundry long, they would be shipped off. So maybe the foundry website should *only* show the "works in progress". How would that be for confusing :-)

**Comment by alanruttenberg** [comment]
I'm thinking primarily the splash page, but with links to the principles and anything else obo foundry specific, e.g. governance, review. The URL for the principles page would be obofoundry, for branding, but the obolibrary site would stay the same, for the most part. The OBO library site might have a nod to the distinction somewhere prominent. Difference between a company home page and catalog. Amazon sells everyone's stuff, but they still let you know, very clearly, what you get for buying from them. Foundry is Amazon Prime ;-) We could take hints from how they do their branding.

**Comment by alanruttenberg** [comment]
I don't think it will be hard to distinguish which content is which. That we distinguish it doesn't mean we have to segregate it. There are other ways to make the distinctions clear (think Prime)

**Comment by mellybelly** [comment]
I just want to avoid folks thinking there are only a few good ontologies in OBO (Library+Foundry). There are many to be reviewed ones that are excellent. And some foundry ones that could use improvement, imho.

**Comment by alanruttenberg** [comment]
I concur with your concern, Melissa. Any way we implement this should be sensitive to the issue and we should have review before implementing to get feedback that we haven't put our foot in our mouth.

**Comment by jamesaoverton** [comment]
Distinguishing the Library from the Foundry and Operations Committee is related to this issue about documentation on the website: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/131

We discussed both issues on the OFOC call 2015-10-06. (OFOC members have access to the call notes.) We will continue the discussion here, with the goal of coming to an actionable proposal by the next meeting. At that time the OFOC can decide whether to put the proposal to a vote.

**Comment by bpeters42** [comment]
My very personal view: As a goal, I would like to see less distinction between the two, which could be achieved by identifying 'universal principles' that everyone agrees with, reviewing all library ontologies towards those, and yes, throwing those out that don't comply (which I am all but certain would primarily be those that are not being maintained). 

I would much rather spend effort on making that happen then on making distinctions between what a foundry vs. a library activity is etc.

**Comment by mellybelly** [comment]
+1

**Comment by zhengj2007** [comment]
+1

**Comment by alanruttenberg** [comment]
Bjoern, the two are not inconsistent. However the the goal you state is, as you put it, a personal view. Doing as you suggest would be a large change from current practice and expectations around the OBO library site. History suggests it would be pushed back against. If we want to head in that direction we want to do so in a deliberative way, not by hanging hopes on a web site change without associated well-thought-out policy. Remember that the Foundry has no formal governance role as far as OBO ontologies in general go. Foundry is an opt-in organization with distinct (and by design opinionated) goals. What we've done has always been with by prior consent.

I am happy to help in efforts to make the larger vision happen. But let's retire the Foundry distinction *after* we've achieved it, rather than before, given the very real possibility that the desired goal might not happen or might need continued existence of a Foundry to make it happen.

**Comment by mcourtot** [comment]
I would prefer to keep only one site. I think we could do better in terms
of organization of that one table, but I don't like the idea of splitting
the content between 2 different domains.
If the current display is an issue could have a Foundry table and a library
table (or headers or..), and clearly describe what each is. There are ways
to differentiate both without going to separate sites IMO.

**Comment by alanruttenberg** [comment]
@mcourtot The goal isn't to strictly separate content. The issue is retaining a distinction between the OBO Foundry and the wider OBO community. All ontologies would still be accessible at obolibrary.org, and most if not all the Foundry content (principles, etc) except that it would be arranged to remove any confusion that the two are the same.

**Comment by alanruttenberg** [comment]
@mcourtot I mean, you and I could both put our CV on a single web page but arrange it so that they are well differentiated. Would you want to do this? If not, why not. If so, well, I'm sorry to hear that ;-)

**Comment by mcourtot** [comment]
@alanruttenberg I'm afraid I don't get the CV analogy, but if you're asking if we were to build a directory of people's CV, I hope you wouldn't suggest to create a website for the french speaking ones, the english speaking ones, another for the senior scientists, another for male and females... Wouldn't you rather have one website and allow users to categorise dynamically according to those criteria of importance to them? If not, well I'm sorry to hear that ;)

I don't think it was suggested we don't retain the distinction between OBO Foundry and the wider OBO community. It doesn't make sense to me to have 2 websites to categorise content, rather than have 2 sections/tables of one site. I thought the whole idea was to homogenise content, while retaining the ability to filter - splitting in 2 different web locations seems counterproductive to me. I'm not really looking for a long argument - we can agree to disagree, make our points and abide by the choice of the majority.

**Comment by alanruttenberg** [comment]
@mcourtot perhaps we differ in the what we perceive as the implications of a "two sites" are. For me the issue is whether the identity and goals of the Foundry are clearly represented, that when someone searches for OBO Foundry, they arrive at a page that is devoted to the OBO Foundry. Typically such a landing page is at the root level for the domain. 

The current domain root shows "Welcome to the OBO Foundry", in red, occupying perhaps 15 percent of the page. It says nothing about what the OBO Foundry is, something that is relegated to menus. It is an easy conclusion (and one that I have verified with a number of colleagues) that the obvious conclusion is that the OBO Foundry product is all the ontologies listed below it's name. That is an incorrect conclusion. Without radical redesign it would be difficult to shake that impression.

While the presence of content labeled as "OBO Foundry" is accessible from the page can be offered as evidence that the identity of the Foundry is not being diminished or extinguished, I am sure that any user study would highlight the same issues I have identified.

For a long time before the site was remodeled this issue about Foundry identity was discussed and various proposals about how to more carefully present the material about the Foundry were on the table. Those discussions, in part, led to our choice of obolibrary.org as the domain for our PURLs. The same rationale is appropriate for the full listing of the ontologies.

While I greatly appreciate the work that has been done in bringing up the new ontology listings, the mere fact of new work does not obviate the issues that we have discussed, and should not lead to a fiat decision on how we present the Foundry to the world.

There is virtually no cost to the "extra site" and much to gain. The cost is an additional repository (cheap) and use of a second domain name that we are *already* using for the project (obolibrary.org). Both development sites would be accessible from the Foundry organization Github page. The same people would edit both. The Foundry site would most likely be a set of static pages, ones that would be focused on the primary goals of the Foundry. The burden of maintenance for maintaining those static pages on a different domain is virtually nil by any metric, for instance in comparison to what is needed to develop the javascript for the library listing.

The OBO Foundry was not conceived as an organization devoted to creating a portal to a bunch of ontologies that are open. I don't remember it ever being proposed that is should be. Yet a casual visitor to our web site (and first impressions matter) would conclude it is. 

The full listing of the ontologies is very useful. The Foundry effort has contributed technology that is applicable to those ontologies in the library that choose to use it. However the Foundry mission is not focused on them and by not making that abundantly clear we do diminish it.

I recognize the concerns of some would have pointed out that the Foundry ontologies are not necessarily the best, and that there are some high quality ontologies despite not having been reviewed in our process. In my view, an appropriate way to discuss this is to question whether the Foundry should continue to exist at all, or whether it should change its goals (substantially) to de-emphasize review and compliance with principles. That's a forthright way of addressing the criticisms, not by (some) choosing (perhaps unconsciously) to de-emphasize the Foundry in the deployment of the new technical work.

**Comment by bpeters42** [comment]
Alan: I agree with all of what you wrote until the last paragraph.

You wrote: "I recognize the concerns of some would have pointed out that
the Foundry ontologies are not necessarily the best, and that there are
some high quality ontologies despite not having been reviewed in our
process. In my view, an appropriate way to discuss this is to question
whether the Foundry should continue to exist at all, or whether it should
change its goals (substantially) to de-emphasize review and compliance with
principles. That's a forthright way of addressing the criticisms, not by
(some) choosing (perhaps unconsciously) to de-emphasize the Foundry in the
deployment of the new technical work."

I would suggest that there is an alternative that I was trying to emphasize
in my previous emails: Let's make the distinction between ontologies that
are OBO foundry principle compliant vs. not compliant clear cut by by
making the principles clear cut, and let's ensure that we can articulate
and justify what the reasons for each principle are. That is what we have
been working towards in the past year in the operations working group, and
we have made a lot of progress.

Emphasizing differences between foundry vs. non-foundry ontologies without
first setting clear criteria comes what distinguishes one from the other
will continue the perception of the OBO foundry acting as an unaccountable
cabal that is picking favorites. And it doesn't help that many current
'foundry' ontologies violate core principles.

Let's get our house in order before bragging about it.

- Bjoern








, and that I and everyone else on the






On Thu, Oct 8, 2015 at 10:30 PM, Alan Ruttenberg <notifications@github.com>
wrote:

> @mcourtot <https://github.com/mcourtot> perhaps we differ in the what we
> perceive as the implications of a "two sites" are. For me the issue is
> whether the identity and goals of the Foundry are clearly represented, that
> when someone searches for OBO Foundry, they arrive at a page that is
> devoted to the OBO Foundry. Typically such a landing page is at the root
> level for the domain.
>
> The current domain root shows "Welcome to the OBO Foundry", in red,
> occupying perhaps 15 percent of the page. It says nothing about what the
> OBO Foundry is, something that is relegated to menus. It is an easy
> conclusion (and one that I have verified with a number of colleagues) that
> the obvious conclusion is that the OBO Foundry product is all the
> ontologies listed below it's name. That is an incorrect conclusion. Without
> radical redesign it would be difficult to shake that impression.
>
> While the presence of content labeled as "OBO Foundry" is accessible from
> the page can be offered as evidence that the identity of the Foundry is not
> being diminished or extinguished, I am sure that any user study would
> highlight the same issues I have identified.
>
> For a long time before the site was remodeled this issue about Foundry
> identity was discussed and various proposals about how to more carefully
> present the material about the Foundry were on the table. Those
> discussions, in part, led to our choice of obolibrary.org as the domain
> for our PURLs. The same rationale is appropriate for the full listing of
> the ontologies.
>
> While I greatly appreciate the work that has been done in bringing up the
> new ontology listings, the mere fact of new work does not obviate the
> issues that we have discussed, and should not lead to a fiat decision on
> how we present the Foundry to the world.
>
> There is virtually no cost to the "extra site" and much to gain. The cost
> is an additional repository (cheap) and use of a second domain name that we
> are *already* using for the project (obolibrary.org). Both development
> sites would be accessible from the Foundry organization Github page. The
> same people would edit both. The Foundry site would most likely be a set of
> static pages, ones that would be focused on the primary goals of the
> Foundry. The burden of maintenance for maintaining those static pages on a
> different domain is virtually nil by any metric, for instance in comparison
> to what is needed to develop the javascript for the library listing.
>
> The OBO Foundry was not conceived as an organization devoted to creating a
> portal to a bunch of ontologies that are open. I don't remember it ever
> being proposed that is should be. Yet a casual visitor to our web site (and
> first impressions matter) would conclude it is.
>
> The full listing of the ontologies is very useful. The Foundry effort has
> contributed technology that is applicable to those ontologies in the
> library that choose to use it. However the Foundry mission is not focused
> on them and by not making that abundantly clear we do diminish it.
>
> I recognize the concerns of some would have pointed out that the Foundry
> ontologies are not necessarily the best, and that there are some high
> quality ontologies despite not having been reviewed in our process. In my
> view, an appropriate way to discuss this is to question whether the Foundry
> should continue to exist at all, or whether it should change its goals
> (substantially) to de-emphasize review and compliance with principles.
> That's a forthright way of addressing the criticisms, not by (some)
> choosing (perhaps unconsciously) to de-emphasize the Foundry in the
> deployment of the new technical work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OBOFoundry/OBOFoundry.github.io/issues/127#issuecomment-146758405>
> .
>



-- 
Bjoern Peters
Associate Professor
La Jolla Institute for Allergy and Immunology
9420 Athena Circle
La Jolla, CA 92037, USA
Tel: 858/752-6914
Fax: 858/752-6987
http://www.liai.org/pages/faculty-peters

**Comment by alanruttenberg** [comment]
@bjoern42,

Glad we're getting into sync. The work that the editorial group has been doing has been a highlight of the activities of the Foundry, in my view, and I think we're in agreement on the goals  regarding principles. 

A couple of clarifications. First, I've tried hard to convey (not quite successfully, apparently :) that my proposal does not try to point out differences between Foundry ontologies versus others. Second, there's nothing in my intention (and I don't think my writing) that is in the direction of bragging or disparaging ontologies. That's not to say I don't think about that general issue, but it's just not been a factor in what I'm proposing.

The focus here is on the difference between portal and organization. With the site, at the moment, there isn't any apparent difference. Even once our house is in order, there will remain a difference between our efforts to help the ontology community in general, such as in providing portal, tech, and advice, and our efforts to focus on a more narrowly delineated set of work. 

The public face of our 'house' is one one of the two sites I've proposed. The point isn't to misrepresent or exaggerate, or alienate. It is to tell our colleagues there is this thing going on, and here is where you go to find out about that thing. Our thing *isn't* the contents of the portal. That's what the current site is. Our thing is  that we are a group of ontology developers that are working hard to advance the field. There's a lot to say about what we are trying to do, including our goals, successes, failure and that's a completely different thing than the specific ontologies that have gone through our review. You might say that the Foundry web site is about the Journey, and the portal is the destination. 

Another view, looking out on the world of companies on the web. Their front page is focused on their organization. They mention their products and they link to catalogs, detail pages and the like. If we followed that design, we would strengthen the presentation of the Foundry as organization, but 

a) The ontologies here aren't our product - the technology to present them, to provide them dereferencing capabilities if they want, etc. is.  Even the 'Foundry' ontologies aren't the product of the Organization - they are other people's work. We contribute. We comment.

b) It's a good thing to have a portal. It's useful with or without the Foundry. It deserves to be it's own thing its site, and be focused on the contents, access, information, etc since that's its prime function, rather than be a link off the Foundry organization web site.

Bottom line, please do not associate my efforts on this issue  as one that is tied to bragging or assertion of superiority or any of that. It hasn't been about that at all (though it is interesting that that is where the conversation happens). It's about good web site design, about making what is presented clear, about recognizing that brand is something valuable but needs to be maintained, and about making a more prominent space for the activities of the Foundry, specifically, but without detracting from the utility of the recent work that has been done to improve the ontology listing, metadata, etc. 

A

**Comment by cmungall** [comment]
OK, back to my proposal

The top level page "obofoundry.org" takes you to a modern looking splash page, e.g something like
https://ropensci.org/

The ontologies listing goes somewhere like "obofoundry.org/ontologies". We make it clear with good old fashioned documentation that this is a motley bunch, some are there for historic reasons, it's an ongoing process, etc.

Does this seem reasonable? We could have "obolibrary.org" redirect straight to the ontologies

**Comment by lschriml** [comment]
Agreed,
  and on the obofoundry.org/ontologies" page, have two headings
for reviewed and not yet reviewed ontologies.
This is an opportunity to point out OBO Foundry principles compliant
ontologies and to encourage groups to apply for review.

Lynn

**Comment by rlwalls2008** [comment]
Yes, sounds like a good plan, for all the reasons mentioned 

Ramona

Enviado de Baja Arizona.

**Comment by alanruttenberg** [comment]
We're quite close. 

But I see no point of having two urls to the ontologies page.  Let it be obolibrary.org alone, let the front page •link• there (there needs to be a link regardless)  and I think we're good. 

I've explained why that's a good ideas. If you disagree, how about explaining why having it be obofoundry.org/ontologies given any benefit. Or why, (and I don't see the point of this, but it is worth understanding the thinking) obofoundry.org/ontologies doesn't redirect to obolibrary.org instead of the other way around. 

Alan

**Comment by bpeters42** [comment]
I prefer the implementation Chris suggested. But also don't feel strong
enough about this to continue arguing. Would be nice to see Alan do the
majority of implementing this either way; my main concern is really wasting
effort on implementing making this distinction rather than all the other
things we could do with more community and scientific benefit.

On Sun, Oct 11, 2015 at 3:11 PM, Alan Ruttenberg <notifications@github.com>
wrote:

> We're quite close.
>
> But I see no point of having two urls to the ontologies page. Let it be
> obolibrary.org alone, let the front page •link• there (there needs to be
> a link regardless) and I think we're good.
>
> I've explained why that's a good ideas. If you disagree, how about
> explaining why having it be obofoundry.org/ontologies given any benefit.
> Or why, (and I don't see the point of this, but it is worth understanding
> the thinking) obofoundry.org/ontologies doesn't redirect to obolibrary.org
> instead of the other way around.
>
> Alan
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OBOFoundry/OBOFoundry.github.io/issues/127#issuecomment-147186729>
> .
>



-- 
Bjoern Peters
Associate Professor
La Jolla Institute for Allergy and Immunology
9420 Athena Circle
La Jolla, CA 92037, USA
Tel: 858/752-6914
Fax: 858/752-6987
http://www.liai.org/pages/faculty-peters

**Comment by alanruttenberg** [comment]
I can implement the domain stuff and take s first pass at the front page, if that's desired.

**Comment by cmungall** [comment]
I'd like to see a clear plan drafted first. I'm not sure I understand 
what is being proposed and what the implications are for complexity, 
future maintainability, etc

**Comment by alanruttenberg** [comment]
Will do

**Comment by alanruttenberg** [comment]
BTW, is there an existing plan of the sort you request available for the current site? Much of what I would document should be already documented there and I'd rather not replicate work.

**Comment by nlharris** [comment]
Should this remain open?

**Comment by bpeters42** [comment]
It was actually 'fun' to re-read essentially the same arguments we are having now being there in 2015. But more importantly, it gave me a sense of pride and accomplishment (yes) to realize we have come a very long way in implementing many of the things discussed back then with the principles review, automation of validation, and the OBO dashboard. 

I think there are no specifics in this issue that are actionable. So I would advocate closing, and anyone wanting to take forward one bit of it should create a new ticket that is more 2020.
